### PR TITLE
Feature: reusable month top users (eg. month top questers)

### PR DESCRIPTION
**Pull request by JakubFrackowiak**
Resolves: #133 

In User component height is fixed to make all users look evenly spaced and the same size. That won't bother the site i belive.

Fullscreen: 
image

Mobile:
image

### CODE REVIEW

File: components/MonthTopUsers/User.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            borderRadius: "50%",
+            objectFit: "cover",
+          }}
+        />
+        <Stack>
+          <Typography variant="h6" fontWeight={500}>
+            {hero.name.first} {hero.name.last}
+          </Typography>
+          <Typography variant="body1" color="primary.main">
+            1,252 XP gained
```
Comment (piotrostr) [inline components/MonthTopUsers/User.tsx:31]: make XP gained props

**Review by piotrostr — CHANGES_REQUESTED** [review]
please also add some padding, it looks a bit packed

**Comment by piotrostr** [comment]
and also please include the reusable buttons, should be merged soon

File: components/MonthTopUsers/User.tsx (hunk 1 above)
Comment (JakubFrackowiak) [reply components/MonthTopUsers/User.tsx]: xpGained props is there, I suppose that it will be fetched from another database.

**Review by piotrostr — APPROVED** [review]
lgtm

**Pull request merged**